django-cloudlaunch/baselaunch/backend_plugins/cloudman_app.py

```diff
@@ -101,8 +101,7 @@ def process_app_config(name, cloud_version_config, credentials, app_config):
             ec2_creds = provider.security.get_or_create_ec2_credentials()
             user_data['access_key'] = ec2_creds.access
             user_data['secret_key'] = ec2_creds.secret
-
-        if hasattr(cloud, 'azure'):
+        elif hasattr(cloud, 'azure'):
```

Comment:
@machristie Thanks for catching this Marcus. I've made a mistake in merging this in. This version of cloudman only supports EC2 clouds, so the original error message is correct. Would you mind deleting the else clauses for both azure and gce and restoring the original error message?